lib/utils/autocomplete.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const tab = require('tabtab')({
+  name: 'serverless',
+});
+
+const autocomplete = () => {
+  tab.on('serverless', (data, done) => {
+    done(null, [
+      'create',
+      'install',
+      'pacakge',
+      'deploy',
+      'invoke',
+      'logs',
+      'login',
+      'metrics',
+      'info',
+      'rollback',
+    ]);
```

Comment:
Coudn't we in general extract the available commands by `serverless.pluginManager.getCommands()`, parse them and build the event consumers accordingly. All options and commands should be available in the retrieved commands array, including active plugins?

Of course this would mean, that the completion registration/creation had to be invoked on every SLS start after the plugins have been loaded.